syntax = "proto3";

option java_multiple_files = true;
option java_package = "org.qsn.api.rpc.grpc.proto.entity.customized";

package org.qsn.protobuf;

import "qsn/entity/common/BInteger.proto";

message TransferTxDataDto{
    bytes to = 1;
    BInteger value = 2;
}